incompatibility/incompatibility-report/incomp-scanning.go

```diff
@@ -17,9 +17,11 @@ import (
 	openapiv3 "github.com/googleapis/gnostic/openapiv3"
 )
 
+// Scan for incompatibilities in an OpenAPI document
 func getIncompatibilites(document *openapiv3.Document) *IncompatibilityReport {
```

Comment:
get can imply that we are simply retrieving data already present.  Perhaps a verb which conveys work is being done to calculate?

Reply:
renamed to ScanIncompatibilities